plat/xilinx/versal/platform.mk

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2013-2016, ARM Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
```

Comment:
You should also migrate to the MULTI_CONSOLE_API.

Reply:
@antonio-nino-diaz-arm  Sure, will do that and will send next version in a week or two.